src/cart/crt_iv.c

```diff
@@ -3270,15 +3575,15 @@ crt_iv_get_nchildren(crt_iv_namespace_t ivns, uint32_t class_id,
 		ivns_internal->cii_gns.gn_tree_topo, root_rank, self_rank,
 		nchildren);
 	if (rc != 0)
-		D_ERROR("crt_tree_get_nchildren(grp %s, root %d self %d), "
-			"failed, rc=%d.\n",
+		D_ERROR("crt_tree_get_nchildren(grp %s, root %d self %d)",
 			ivns_internal->cii_grp_priv->gp_pub.cg_grpid,
-			root_rank, self_rank, rc);
-
+			root_rank, self_rank);
+		D_ERROR("failed, rc=%d.\n", rc);
```

Comment:
why is rc printed separately here? we want it to be printed as part of the original error message